src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -7,7 +7,10 @@ import sinon from 'sinon';
 import { fromUnixTime } from 'date-fns';
 import { format } from 'date-fns-tz';
 
-import { VA_FORM_IDS } from 'platform/forms/constants';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import { VA_FORM_IDS } from '~/platform/forms/constants';
```

Comment:
For this one can we import from `'../../constants';` like the pattern I've see you update in other files since we're inside platform here.

Reply:
I like to be inconsistent 😉 LOL